frontend/awx/resources/sources/InventorySourceForm.tsx

```diff
@@ -131,14 +133,12 @@ export function EditInventorySource() {
     () => ({
       name: inventorySource?.name,
       description: inventorySource?.description ?? '',
+      execution_environment: inventorySource?.summary_fields?.execution_environment?.name,
       source: inventorySource?.source,
       credential: inventorySource?.summary_fields?.credential?.name,
       source_project: inventorySource?.summary_fields?.source_project,
       source_path: {
-        name:
-          inventorySource?.source_path?.length === 0
-            ? '/ (project root)'
-            : inventorySource?.source_path,
+        name: inventorySource?.source_path,
```

Comment:
Given that this is in edit mode I do think we want to keep this line the way it was.  Inventory file is a required field and so as a user I would expected that field to be populated on when the form loads
![Screenshot 2024-05-01 at 1 17 01 PM](https://github.com/ansible/ansible-ui/assets/39280967/94f7e4fb-9de4-40ea-ad91-d1557dac5c61)

Reply:
I'm not sure I follow. If you're editing a source that already has an inventory file, it will load the file path. If you're creating a new source or changing the source type from something else to project, that field will be blank as there is no existing file chosen yet.

Reply:
whoops, you are right.  I assumed the api worked a bit differently with the inventory file.